### DISCUSSION TITLE
Signed into personal acc instead of work acc

### DISCUSSION THREAD

**Discussion by Total-Gobshite** [community/community#62738] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I messed up. Signed into the wrong account.

Work recently gave devs GitHub Copilot, when signing into the extension, I didn't realise I'd signed in using my personal account (which also has Copilot). 

How bad have I messed up? 

I think I had 3 files open on the work repo (which is private). Realised 30 minutes later, came back to see my personal account signed into vscode and changed that.

Is there any way I can request to delete any data which may have been shared?

**Comment by Akash1134** [comment, marked as answer]
Hi @Total-Gobshite ,

Depending on the [telemetry settings](https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-settings-on-githubcom) for your Copilot for Individual account (if you have the data transmission enabled), there are chances you might have transmitted prompts or suggestions only if both your business and personal accounts are associated with same GitHub ID.

We recommend that you keep Copilot for Business and Copilot for Individual accounts linked to separate GitHub accounts (dotcom user).

If you believe data was transmitted while you were logged in with your personal account, you can request for a data deletion: If you wish to delete Prompts and Suggestions associated with your GitHub identity, contact [GitHub Support](https://support.github.com/contact?tags=docs-generic).

Thanks!

**Reply by Total-Gobshite** [reply]
Fantastic response @Akash1134, thank you kindly! Luckily I had telemetry settings unchecked and both accounts aren't linked to the same GH ID.
